src/shell/data-staging.cpp

```diff
@@ -197,30 +220,36 @@ static int main_init (flux_plugin_t *p)
 
     json info;
     try {
-        info = json::parse(info_cstr);
+        info = json::parse (info_cstr);
     } catch (json::parse_error& e) {
-        shell_log_error ("Error parsing shell \"info\" json at byte %zu. %d: %s", e.byte, e.id, e.what());
+        shell_log_error ("Error parsing shell \"info\" json at byte %zu. %d: %s",
+                         e.byte, e.id, e.what ());
         errno = EINVAL;
         return -1;
     }
 
     int rank;
     json jobspec, R;
     try {
-        rank = info["rank"].get<int>();
-        jobspec = info["jobspec"];
-        R = info["R"];
+        rank = info.at ("rank").get<int>();
+        jobspec = info.at ("jobspec");
+        R = info.at("R");
```

Comment:
Nit: space before parenthesis.
```suggestion
        R = info.at ("R");
```